engine/access/rpc/engine.go

```diff
@@ -27,27 +28,31 @@ import (
 
 // Config defines the configurable options for the access node server
 type Config struct {
```

Comment:
I think we want to add a comment on what "Secure" precisely means here (i.e. self-signed TLS + Authentication through pre-shared keys)

Reply:
added a line for it.